spec/stac-geoparquet-spec.md

```diff
@@ -0,0 +1,39 @@
+# STAC GeoParquet Specification
+
+## Overview
+
+This document specifies how to map a set of [STAC Items](https://github.com/radiantearth/stac-spec/tree/v1.0.0/item-spec) into
+[GeoParquet](https://geoparquet.org). It is directly inspired by the [STAC GeoParquet](https://github.com/stac-utils/stac-geoparquet)
+library, but aims to provide guidance for anyone putting STAC data into GeoParquet. 
+
+## Guidelines
+
+Generally most all the fields in a STAC Item should be mapped to a row in GeoParquet. We embrace Parquet structures where possible, mapping
+from JSON into nested structures. We do pull the properties to the top level, so that it is easier to query and use them. The names of the
+most of the fields should be the same in STAC and in GeoParquet.
+
+| Field           | GeoParquet Type    | Required | Details                                            |
+| --------------- | ------------------ | ---------|--------------------------------------------------- |
+| type            | String             | Optional | This is just needed for GeoJSON, so it is optional and not recommended to include in GeoParquet |
+| stac_extensions | List of Strings    | Required | This column is required, but can be blank if no STAC extensions were used |
```

Comment:
Yeah I think `[]` is ideal. Are there going to be cases where "I don't know if there were extensions or not" is true? I figure the original Item will always have this array, even if it's empty?

Reply:
Yeah, I agree that `[]` is good. 

The `stac_extensions` field is not required, so the array won't always be there.

Reply:
Oh I see; it's not required in the STAC JSON. We may want the column to _exist_ even if most rows are null?

One thing the reader will have to do is know which fields the STAC spec _may_ have at the top level, so that it knows what not to wrap in a `properties` key

Reply:
Yeah, I think it is good to have the column to exist - put it as required.

That is a good point on the top level keys - we probably should have some explicit guidance on what to do if you have a property that conflicts with a top-level name. I don't think we have restrictions on people naming their properties `assets`, `id`, `links`. It'd certainly be confusing. Maybe we recommend that if people have that then they name the top level field `properties_id` or something, and the actual top level stac one takes precedence?

Reply:
If we're planning to usually bring properties up to the top level, I don't think there's a good suggestion to be made with what to do with column names. I don't want to be renaming columns unexpectedly, or even having different behavior of whether to bring names to the top level if there's a name overlap.

So I'd suggest the spec say: "STAC GeoParquet does not support properties that are named such that they collide with a top-level key". The code should error in this case.